package/src/prmon.cpp

```diff
@@ -158,7 +189,11 @@ int MemoryMonitor(pid_t mpid, char* filename, char* jsonSummary, unsigned int in
 	       << valuesIO[0]   << "\t" 
 	       << valuesIO[1]   << "\t" 
 	       << valuesIO[2]   << "\t" 
-	       << valuesIO[3]   << std::endl;  
+	       << valuesIO[3]   << "\t"
```

Comment:
Divide by `clock_ticks` here instead (with conversion to `float` or `double`).

Reply:
BTW I think your comment was for the `valuesCPU` bit and not the `valuesIO`.

Reply:
Yes! Sorry for any confusion.